content/en/docs/Installation/installation-guide/accessing-kiali.md

```diff
@@ -42,56 +42,78 @@ have the necessary privileges to perform port forwarding.
 
 ## Accessing Kiali through an Ingress {#ingress-access}
 
-By default, the installation exposes Kiali creating an
-[Ingress resource](https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/templates/kubernetes/ingress.yaml).
-Find out your Ingress IP or domain name and use it to access Kiali by
-visiting this URL in your web browser:
-`https://_your_ingress_ip_or_domain_/kiali`.
+You can configure Kiali to be installed with an
+[Ingress resource](https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/templates/kubernetes/ingress.yaml)
+defined, allowing you to access
+the Kiali UI through the Ingress. By default, an Ingress will not be created. You can
+enable the Ingress by setting `spec.deployment.ingress.enabled` to `true` in the Kiali
+CR (a similar setting for the server Helm chart is available if you elect to install Kiali
+via Helm as opposed to the Kiali Operator).
```

Comment:
Maybe we can add here that this is more of an example than a reference? Exposing Kiali externally through the `spec.deployment.ingress` is a convenient way of exposing Kiali externally but it's not necessarily the recommended way because that'll highly depend on your environment and how services are exposed generally for your environment.

Reply:
@nrfox see my update with this verbiage: https://deploy-preview-466--kiali.netlify.app/docs/installation/installation-guide/accessing-kiali/#ingress-access

Reply:
Great!